doc/source/whatsnew/v0.20.0.txt

```diff
@@ -484,6 +484,7 @@ Deprecations
 - ``DataFrame.astype()`` has deprecated the ``raise_on_error`` parameter in favor of ``errors`` (:issue:`14878`)
 - ``Series.sortlevel`` and ``DataFrame.sortlevel`` have been deprecated in favor of ``Series.sort_index`` and ``DataFrame.sort_index`` (:issue:`15099`)
 - importing ``concat`` from ``pandas.tools.merge`` has been deprecated in favor of imports from the ``pandas`` namespace. This should only affect explict imports (:issue:`15358`)
+- ``DataFrame.consolidate()`` has been deprecated. (:issue:`15483`)
```

Comment:
say ``Series/DataFrame/Panel.consolidate()`` it affects everything

say has been deprecated as a public method as